bindings/python/scripts/generate.py

```diff
@@ -144,7 +145,7 @@ def get_fields_from_anonymous(self, item):
         fields = []
         for sub_item in item["members"]:
             if sub_item["kind"] in ("ANONYMOUS_UNION_DECL", "ANONYMOUS_STRUCT_DECL"):
-                for field in self.get_fields_from_anonymous(sub_item):
+                for field in bind.get_fields_from_anonymous(sub_item):
```

Comment:
`bind` is a reserved word in Python?

Reply:
I don't think so
```py
>>> import keyword
>>> for k in keyword.kwlist:
...     print(k)
... 
False
None
True
and
as
assert
break
class
continue
def
del
elif
else
except
finally
for
from
global
if
import
in
is
lambda
nonlocal
not
or
pass
raise
return
try
while
with
yield


```

Reply:
Oh cool. PS: why change from `self` to `bind`?

Reply:
Changed to `@staticmethod` as suggested by you, so no `self`.

Reply:
```python
class A:
    def __init__(self):
        self.say_hi()
    @staticmethod
    def say_hi():
        print("I'm a static method, short and stout")

A()
A.say_hi()
```